libseccomp/src/lib.rs

```diff
@@ -41,25 +41,40 @@
 //! ```
 
 pub mod error;
+#[cfg(libseccomp_v2_5)]
 pub mod notify;
 
 use error::ErrorKind::*;
 use error::{Result, SeccompError};
 use libseccomp_sys::*;
 use std::convert::TryInto;
 use std::ffi::{CStr, CString};
+use std::fmt;
 use std::os::unix::io::AsRawFd;
 use std::ptr::NonNull;
 
-/// ScmpVersion represents the version information of
-/// the currently loaded libseccomp library
+/// Represents the version information of the libseccomp library.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
 pub struct ScmpVersion {
     pub major: u32,
     pub minor: u32,
     pub micro: u32,
 }
 impl ScmpVersion {
+    /// Creates a `ScmpVersion` from the specified arbitrary version.
+    ///
+    /// # Arguments
+    ///
+    /// * `version` - A tuple that represents the version of the libseccomp library.  
+    /// The index 0, 1, and 2 represent `major`, `minor`, and `micro` respectively.
+    pub fn from(version: (u32, u32, u32)) -> Self {
```

Comment:
We could move this into an `impl From<(u32, u32, u32)> for ScmpVersion`.

Reply:
I forgot to use `impl From`. Thank you! I fixed it.